### ISSUE TITLE
Direct Deposit v2 - focus is on Edit button on first launch

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#9075]
When first going to the new Direct Deposit feature, the focus should be on the section headline, but instead it is on the Edit button.

![image.png](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/d28be22e-cb63-4034-8e5c-96f3c337a0b5)

**Comment by erikphansen** [comment]
![dd-focus](https://user-images.githubusercontent.com/20728956/81854234-7a2de700-9512-11ea-8b25-9f5575a0edec.gif)

**Comment by erikphansen** [comment]
@mshea0606 can you confirm that the Edit Direct Deposit button does not get focus when the section loads? You can test with user 36 in staging.

**Comment by erikphansen** [comment]
@mshea0606 thanks for pointing out that this feature crashes on staging right now. I made a ticket for that: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9819

**Comment by mshea0606** [comment]
@erikphansen this looks good
